### ISSUE TITLE
Gallery issues 1.1.18 (it also had something similar in 1.1.17)

### ISSUE THREAD

**Issue by paaljoachim** [wplemon/gridd#113]
Hey Ari

I have a feeling I have mentioned this earlier, but was not able to locate it.
Here is an example of a 3 image gallery showing the backend:

![Screen Shot 2019-10-14 at 12 25 09](https://user-images.githubusercontent.com/5323259/66767211-e4e44a80-ee7d-11e9-839c-a6e03b5bf1d6.png)

Then the frontend:
Zoomed out view.
![Screen Shot 2019-10-14 at 12 27 10](https://user-images.githubusercontent.com/5323259/66767284-06ddcd00-ee7e-11e9-870c-54f3cb5f7671.png)

Then normal 100% view:
![Screen Shot 2019-10-14 at 12 27 23](https://user-images.githubusercontent.com/5323259/66767305-0f360800-ee7e-11e9-91cf-d1711e45869d.png)

So it seems like the gallery backend -> frontend is a bit off.

**Comment by aristath** [comment]
I can't replicate on the dev branch, so I'm assuming this has already been fixed by one of the dozens of commits that were pushed for v1.2.

Closing this one, if during the beta-testing period we find this is an issue we can reopen.

I tested using the [WPTRT Unit Tests XML](https://raw.githubusercontent.com/WPTRT/theme-unit-test/master/themeunittestdata.wordpress.xml) which has lots of test cases for the gallery block.
